docs/datadog_dashboard.md

```diff
@@ -0,0 +1,82 @@
+# Datadog Dashboards
+This feature will be introduced in DatadogOperator v1.9.0 and is currently in beta.
+
+## Overview
+The Datadog Dashboards CRD allows users to create dashboards using the operator and manage them as Kubernetes resources. 
```

Comment:
What does CRD stand for? Or is this a commonly used acronym?

Reply:
Custom Resource Definition - it's commonly used and well understood acronym for Kubernetes users.